src/components/DevDummyLogin/index.js

```diff
@@ -192,6 +207,8 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
     if (savedPasswordText) setPasswordText(savedPasswordText);
   }, []);
 
+  console.log(dummyInput);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This console.log statement should be removed from production code. It's generally not a good practice to leave debugging logs in the codebase as they can expose sensitive information.

```diff
-  console.log(dummyInput);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->